cassandra/src/main/scala/filodb.cassandra/columnstore/PartitionKeysTable.scala

```diff
@@ -0,0 +1,69 @@
+package filodb.cassandra.columnstore
+
+import java.lang.{Integer => JInt, Long => JLong}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{ExecutionContext, Future}
+
+import com.datastax.driver.core.ConsistencyLevel
+import monix.reactive.Observable
+
+import filodb.cassandra.FiloCassandraConnector
+import filodb.core.{DatasetRef, Response}
+import filodb.core.store.PartKeyRecord
+
+sealed class PartitionKeysTable(val dataset: DatasetRef,
+                                val shard: Int,
+                                val connector: FiloCassandraConnector,
+                                writeConsistencyLevel: ConsistencyLevel)
+                               (implicit ec: ExecutionContext) extends BaseDatasetTable {
+
+  import filodb.cassandra.Util._
+
+  val suffix = s"partitionkeys_$shard"
+
+  val createCql =
+    s"""CREATE TABLE IF NOT EXISTS $tableString (
+       |    partKey blob,
+       |    startTime bigint,
+       |    endTime bigint,
+       |    PRIMARY KEY (partKey)
+       |) WITH compression = {'sstable_compression': '$sstableCompression'}""".stripMargin
+
+  lazy val readCql = s"SELECT segmentid, segment " +
```

Comment:
What is this for?

Reply:
good catch. Remnant from old code. Removable.